worldengine/cli/main.py

```diff
@@ -441,10 +450,10 @@ def main():
     if args.temps and len(args.temps.split('/')) is not 6:
         usage(error="temps must have exactly 6 values")
 
-    temps =[.874, .765, .594, .439, .366, .124]
+    temps = [.874, .765, .594, .439, .366, .124]
     if args.temps:
         temps = args.temps.split('/')
-        for x in range(0,6):
+        for x in range(0,len(temps)):
```

Comment:
I think that also issues in the order or the value (range 0..1) could lead to crashes or at least unpredictable behavior later. The best person for confirming/denying this is probably @esampson 

I do not think we should use an assert but instead handle the errors printing a nice message and closing the application with an error code.